pkg/karmadactl/cmdinit/cmdinit.go

```diff
@@ -115,6 +118,7 @@ func NewCmdInit(parentCommand string) *cobra.Command {
 	// karmada
 	crdURL := fmt.Sprintf("https://github.com/karmada-io/karmada/releases/download/%s/crds.tar.gz", releaseVer.FirstMinorRelease())
 	flags.StringVar(&opts.CRDs, "crds", crdURL, "Karmada crds resource.(local file e.g. --crds /root/crds.tar.gz)")
+	flags.StringVarP(&opts.KarmadaAPIServerAdvertiseAddress, "karmada-apiserver-advertise-address", "", "", "The IP address the Karmada API Server will advertise it's listening on. If not set the host cluster master IP address will be used.")
```

Comment:
> If not set the host cluster master IP address will be used.

I'm not sure it's a valid sentence :) (And sorry for I didn't point it out in the last review)

How about change to `If not set, the address on the master node will be used.`

Reply:
I think I should also change in the `help prompt`  Do you think so ?

Reply:
yes.

Reply:
updated